llmfoundry/data/finetuning/dataloader.py

```diff
@@ -171,7 +180,25 @@ def build_finetuning_dataloader(
         given a starting workload YAML.
     """
     dataset_cfg = dataset
-    _validate_config(**dataset_cfg)
+    is_streaming = dataset_cfg.get('remote') is not None or dataset_cfg.get(
+        'streams',
+    ) is not None
```

Comment:
```suggestion
    is_streaming = (dataset_cfg.get('remote') is not None or dataset_cfg.get('streams') is not None)
```
I think reformatting this will give you better wrapping?